Xamarin.Forms.Platform.Android/Renderers/ShellItemRenderer.cs

```diff
@@ -419,14 +419,7 @@ void UpdateTabBarVisibility()
 			if (DisplayedPage == null)
 				return;
 
-			bool visible = ShellItemController.ShowTabs;
```

Comment:
You say you're fixing a bug for iOS, so what is this code change doing here?

Reply:
The point of `ShowTabs` was to move all the tab display logic up into the xplat code so we don't have to duplicate it across the renderers. I noticed this code on the android side when verifying that it matches up with iOS

It's currently a noop since the logic is taken care of by `showtabs` but since it's duplicated logic this change avoids 5 months from now when I modify the behavior of `showtabs` but android for some reason works differently :-)